app/assets/javascripts/admin/addon/controllers/admin-users-list-show.js

```diff
@@ -1,25 +1,38 @@
+import { tracked } from "@glimmer/tracking";
```

Comment:
I think there's a lot that can be done in terms of code organization here, but since this is still a controller, we can tackle that separately as a modernization project. 🙂

Reply:
Yeah I agree. I think ideally this controller should be just a wrapper for a component that contains all the logic that currently exists in this controller, but I didn't want to include a fairly large refactoring like that in this PR.